### ISSUE TITLE
slice-copy interp test fails with Go 1.16 / LLVM 13

### ISSUE THREAD

**Issue by QuLogic** [tinygo-org/tinygo#2836]
I'm not sure which of these is the issue, as we don't have a system with the intermediate combinations. 

When building on Fedora 36, everything passes with Go 1.18 and LLVM 14. When building on Fedora 35 with Go 1.16 and LLVM 13, the slice-copy test fails, and the diff is:
```diff
diff --git a/interp/testdata/slice-copy.out.ll b/interp/testdata/slice-copy.out.ll
index 28175643..9f88df09 100644
--- a/interp/testdata/slice-copy.out.ll
+++ b/interp/testdata/slice-copy.out.ll
@@ -1,6 +1,8 @@
 target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
 target triple = "x86_64--linux"
 
+@"main$alloc.1" = internal unnamed_addr constant [6 x i8] c"\05\00{\00\00\04", align 8
+
 declare void @runtime.printuint8(i8) local_unnamed_addr
 
 declare void @runtime.printint16(i16) local_unnamed_addr
@@ -15,6 +17,7 @@ entry:
   call void @runtime.printuint8(i8 3)
   call void @runtime.printuint8(i8 3)
   call void @runtime.printint16(i16 5)
-  call void @runtime.printint16(i16 5)
+  %int16SliceDst.val = load i16, i16* bitcast ([6 x i8]* @"main$alloc.1" to i16*), align 2
+  call void @runtime.printint16(i16 %int16SliceDst.val)
   ret void
 }
```
No other interp tests appear to fail, and all other tests pass otherwise.

**Comment by QuLogic** [comment]
Looking at the log, I see this file was changed in 6b31ee1e9392a463b536735081cc496405929882 for LLVM 14 support. Should this test be skipped on older LLVM then?

**Comment by QuLogic** [comment]
Fixed by #2837.

**Comment by deadprogram** [comment]
Reopening and tagging to close on next release.
